lib/net/imap.rb

```diff
@@ -443,6 +443,11 @@ def login(user, password)
     #
     # A Net::IMAP::NoResponseError is raised if the mailbox does not
     # exist or is for some reason non-selectable.
+    #
+    # If the server supports the UIDPLUS extension it may return an additional
+    # "NO" response with a "UIDNOTSTICKY" response code indicating that the
+    # mailstore does not support persistent UIDs:
+    # +@responses["NO"].last.code.name == "UIDNOTSTICKY"+
```

Comment:
n.b. the simple rdoc `+` tends to not work over multiple words.  You need the `<code>` tag version for that.  More importantly, you'll get ruby syntax highlighting if you just indent it. 🙂

```suggestion
    #   @responses["NO"].last.code.name == "UIDNOTSTICKY"
```

Reply:
Ah yes, actually looked up the generated rdoc and also found some problems in the existing comment and fixed that as well.